CI Coq build fails: '-R . Hello' grafts opam stdlib under Hello prefix and breaks Corelib.Prelude lookup

## Symptom

Latest CI run [24429911135](https://github.com/rhencke/orly/actions/runs/24429911135/job/71371871382) fails with:

```
Warning: /home/runner/work/orly/orly/_opam/lib/coq/theories was previously
bound to Corelib; it is remapped to Hello._opam.lib.coq.theories
[overriding-logical-loadpath,filesystem,default]
Error: Cannot find a physical path bound to logical path Prelude with prefix Corelib.
make: *** [Makefile:4: build] Error 1
```

## Root cause

`_CoqProject` and the `Makefile` both pass `-R . Hello` — recursive include of the current directory under the `Hello` prefix.  Locally the project root only has `Hello.v`, but in CI `opam install -y coq` puts the entire Coq stdlib into `./_opam/lib/coq/theories` *inside the project root*.  The recursive `-R .` then remaps every stdlib module under `Hello.`, replacing the canonical `Corelib.` namespace.  When `Hello.v` does `From Corelib Require Prelude`, Coq can't find it anymore.

## Fix

Exclude `_opam` from the recursive include.  Two options:

1. **Replace recursion with explicit file list** in `_CoqProject`:

   ```
   -Q . Hello
   Hello.v
   ```

   `-Q` is non-recursive — only files in this exact directory.  Same effect for a one-file project, no chance of remapping anything in subdirs.

2. **Move sources into a subdirectory** like `src/` and use `-R src Hello` so opam's install path stays out of the include scope.

Also update the `Makefile` build target to use `-Q` (or `coq_makefile -f _CoqProject -o Makefile.coq` and delegate, the standard Coq pattern).

(1) is the smallest fix; (2) sets up better as the project grows beyond one file.

--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 .PHONY: build test clean
 
 build:
-	rocq compile -R theories Hello theories/Hello.v
+	rocq compile -Q theories Hello theories/Hello.v
 
 test: build
 

--- a/_RocqProject
+++ b/_RocqProject
@@ -1,3 +1,3 @@
--R theories Hello
+-Q theories Hello
 
 theories/Hello.v

